pytest_pootle/fixtures/models/store.py

```diff
@@ -370,14 +370,6 @@ def ru_tutorial_po(settings, russian_tutorial):
 
 
 @pytest.fixture
-def ru_update_save_changed_units_po(settings, russian_tutorial):
-    """Require the /ru/tutorial/tutorial.po store."""
-    return _require_store(russian_tutorial,
-                          settings.POOTLE_TRANSLATION_DIRECTORY,
-                          'update_save_changed_units.po')
```

Comment:
currently it is used - but if there are significant benefit to removing if that wasnt the case - the fiels in the tp folders are copied on every test